configure/otel-collector/otel-collector.ConfigMap.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: otel-collector
+  labels:
+    deploy: sourcegraph
+    app.kubernetes.io/component: otel-collector
+    sourcegraph-resource-requires: no-cluster-admin
+data:
+  config.yaml: |
+    receivers:
+      otlp:
+        protocols:
+          grpc: # port 4317
+          http: # port 4318
```

Comment:
Should we note that these (and certain other things) should not be changed?

Reply:
Agreed. That being said, how is this working? I see a comment character here 🤔

Reply:
Fixed. I explained this to you on our call, but for visibility: empty values here default to those ports :) clarified it a bit